packages/mini-browser/src/browser/mini-browser-frontend-module.ts

```diff
@@ -68,6 +76,8 @@ export default new ContainerModule(bind => {
     bind(LocationMapper).to(FileLocationMapper).inSingletonScope();
     bind(LocationMapper).to(HttpLocationMapper).inSingletonScope();
     bind(LocationMapper).to(HttpsLocationMapper).inSingletonScope();
+    bind(LocationMapper).to(IPAddressLocationMapper).inSingletonScope();
+    bind(LocationMapper).to(LocalhostLocationMapper).inSingletonScope();
```

Comment:
could you bind this to itself first and then to service, so we can override it in a cloud scenario automatically mapping to the correct host url?